app/DynamicsAdapter/DynamicsAdapter.Web/Startup.cs

```diff
@@ -57,7 +68,7 @@ public void ConfigureSearchApi(IServiceCollection services)
         /// Configures the Quartz Hosted Service.
         /// </summary>
         /// <param name="services"></param>
-        public void ConfigureScheduler(IServiceCollection services)
+        public void ConfigureScheduler(IServiceCollection services, AppSettings settings)
         {
 
             var schedulerConfiguration = Configuration.GetSection("Scheduler").Get<SchedulerConfiguration>();
```

Comment:
We need to remove that line, as it is passed as a parameter

Reply:
I forgot to hook it up in the previous one lol

Reply:
Where was it passed as parameter ? i did not see that ..

Reply:
I was grabbing it here: ``` var schedulerConfiguration = Configuration.GetSection("Scheduler").Get<SchedulerConfiguration>();```